setup.py

```diff
@@ -32,6 +32,8 @@
 
 packages = [
     'oauth2client',
+    'oauth2client.contrib',
```

Comment:
@trcook If you want to see the code as it was then you can go to 

https://github.com/google/oauth2client/releases

and then click the one you want, e.g.

https://github.com/google/oauth2client/releases/tag/v1.5.2

Check out the `oauth2client` dir then, no `contrib` just yet:
https://github.com/google/oauth2client/tree/73d9d55447de97dfe541395817a0c8241701f7d6/oauth2client